src/tab-page/TabPage.jsx

```diff
@@ -45,7 +45,7 @@ function TabPage({ intl, ...props }) {
         <PageLoading
           srMessage={intl.formatMessage(messages.loading)}
         />
-        <Footer />
```

Comment:
Can we directly use the env variable in the Footer, instead of passing the logo in props?

Reply:
Actually that's how open edx designed it. Also isn't it good practice to store the variable in one place instead of storing in every component?

Reply:
Yup, but we can minimize the core changes by doing this.